tools/ds-identify

```diff
@@ -1409,33 +1376,31 @@ dscheck_VMware() {
 
     # Check to see if running in a container and the VMware
     # datasource is configured via environment variables.
-    if vmware_has_envvar_vmx_guestinfo; then
-        if vmware_has_envvar_vmx_guestinfo_metadata || \
-            vmware_has_envvar_vmx_guestinfo_userdata || \
-            vmware_has_envvar_vmx_guestinfo_vendordata; then
+    if [ -n "${VMX_GUESTINFO}" ]; then
+        if [ -n "${VMX_GUESTINFO_METADATA}" ] || \
+           [ -n "${VMX_GUESTINFO_USERDATA}" ] || \
+           [ -n "${VMX_GUESTINFO_VENDORDATA}" ]; then
             return "${DS_FOUND}"
         fi
     fi
 
     # Do not proceed unless the detected platform is VMware.
-    if [ ! "${DI_VIRT}" = "vmware" ]; then
-        return "${DS_NOT_FOUND}"
-    fi
+    [ "${DI_VIRT}" = "vmware" ] || return ${DS_NOT_FOUND}
```

Comment:
Please change this back as I wanted the logic to be more clear instead of the syntactic sugar approach. Thanks.